ast_canopy/cpp/cmake/ast_canopy_llvm.cmake

```diff
@@ -0,0 +1,51 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+function(ast_canopy_check_llvm_linkage library_target)
+  if(NOT DEFINED LLVM_LINKAGE)
```

Comment:
`LLVM_LINKAGE` seems undocumented? Is there no public option to allow differentiating between shared vs static libraries of an already built LLVM / Clang?